pkg/plugins/hybrid/v1alpha/scaffolds/init.go

```diff
@@ -120,21 +119,30 @@ func (s *initScaffolder) Scaffold() error {
 
 	err = scaffold.Execute(
 		&templates.Main{},
-		&templates.GoMod{ControllerRuntimeVersion: golangv4.ControllerRuntimeVersion},
+		// TODO: This used to be golangv4.ControllerRuntimeVersion to be in sync
+		// with the version scaffolded by the version of Kubebuilder we depend on.
+		// As of writing this comment, Kubebuilder is behind on the kube bumps
+		// so we are manually setting this to the needed controller-runtime version
+		// to facilitate the bump to support k8s 1.30
+		&templates.GoMod{ControllerRuntimeVersion: "v0.18.2"},
 		&templates.GitIgnore{},
 		&templates.Watches{},
 		&rbac.ManagerRole{},
 		&templates.Makefile{
-			Image:                    imageName,
-			KustomizeVersion:         kustomizev2.KustomizeVersion,
-			HybridOperatorVersion:    hybridOperatorVersion,
-			ControllerToolsVersion:   golangv4.ControllerToolsVersion,
-			ControllerRuntimeVersion: golangv4.ControllerRuntimeVersion,
+			Image:                 imageName,
+			KustomizeVersion:      kustomizev2.KustomizeVersion,
+			HybridOperatorVersion: hybridOperatorVersion,
+			// TODO: This used to be golangv4.ControllerRuntimeVersion and golangv4.ControllerToolsVersion
+			// to be in sync with the version scaffolded by the version of Kubebuilder we depend on.
+			// As of writing this comment, Kubebuilder is behind on the kube bumps
+			// so we are manually setting this to the needed controller-runtime version
+			// to facilitate the bump to support k8s 1.30
+			ControllerToolsVersion:   "v0.15.0",
+			ControllerRuntimeVersion: "v0.18.2",
```

Comment:
Thinking about this - wondering if we need to wait till KB releases a 1.30? KB recently has had a few breaking changes, so when we bump to a version of KB that has equivalent k8s 1.30 - would the scaffolding also change?

Reply:
I think we control all the scaffolding logic for this plugin so I don't think so.

Reply:
I take my comment back - realised that this is for Rukpak, so it's not right to depend on SDK. But still worried about scaffolding issues though - wondering if we can skip bumping the scaffolding and just bump the library instead?

Reply:
> I think we control all the scaffolding logic for this plugin so I don't think so.

Yes, but the hybrid plugin also scaffolds the go one - so we are basically going to scaffold out a different k8s version for other dependencies without bumping the relevant KB?

Reply:
Ah I missed that. I can revert the scaffolding changes and capture it in a follow up if that is acceptable?

Reply:
Yeah, sounds good

Reply:
Changes reverted in https://github.com/operator-framework/helm-operator-plugins/pull/337/commits/70c5aaaf1998f3504ca8f257b21d0327382f8b3f